### PR TITLE
Add task list params

### PR DESCRIPTION
**Pull request by jpbruinsslot**
This PR implements a partial scans/tasks lists view, that (according to designs) can be used on: ooi detail page, boefje detail page, and task/scans page. Since, the designs show that the functionality is the same across these pages, it made sense to create one partial and include these in the templates. At the moment in this PR it is implemented on the ooi detail and plugin detail page.

Open questions:

* Since there are different strategies for implementing forms/pagination throughout rocky the main question is if this implementation and location of files etc. is the way forward and correct for the future. With this PR I don't want to introduce yet another new implementation.

* In the designs, the table's first column is the object name. It might not make sense in the context of an ooi detail page, since the name is already present. Additionally, there is a search field for an object name, in the context of a ooi detail this would make not much sense.

* Now form fields are being passed to the template context. This is done to make a differentiation between fields in the filter form fields that need to remain as query parameters when submitting the filter form. This could potentially be a django form, the question is if this is desirable.

How to test/view:
* In mula checkout https://github.com/minvws/nl-kat-mula/pull/16
* Go through the onboarding
* Add an url and do a regular scan
* Choose an object, and scroll down to see the scan history
* Go to kat-alogus
* Select plugin dns-records

### CODE REVIEW

**Comment by Lisser** [comment]
For some reason, the first column remains empty for me. Even after starting from scratch with `make kat`
Screenshot 2023-01-06 at 14 36 45

**Comment by jpbruinsslot** [comment]
@Lisser commit in the mula branch should fix this problem

**Comment by Lisser** [comment]
Awesome. Works for me now!

**Comment by Lisser** [comment]
Can we use a lazy Paginator object here to apply pagination in the same style as the OOIList paginator?

https://github.com/minvws/nl-kat-rocky/blob/main/rocky/views/mixins.py#L153

File: rocky/scheduler.py [hunk 1]
```diff
@@ -115,9 +116,28 @@ def __init__(self, base_uri: str):
         self.session = requests.Session()
         self._base_uri = base_uri
 
-    def list_tasks(self, queue_name: str, limit: int) -> PaginatedTasksResponse:
-        params = {"limit": limit}
-        res = self.session.get(f"{self._base_uri}/schedulers/{queue_name}/tasks", params=params)
+    def list_tasks(
+        self,
+        scheduler_id: str,
+        type: Optional[str] = None,
```
Comment (Lisser) [inline rocky/scheduler.py:122]: Be careful: `type shadows built-in name 'type'`

Best to name it differently: e.g. `type_`, `object_type` or `ooi_type`

**Pull request merged**